### PR TITLE
adding GEONOR and KSS to collection keywords

### LINKED ISSUE #251: Need for new collections
We will need two more collection keywords. One for Klimakverna/KSS (I will discuss this in the project) and one to filter data going towards csw serving genorge.

### PR DESCRIPTION
**Pull request by ferrighi**
Support for filtering of data going towards genorge and additional tags for KSS. This Closes #251

### CODE REVIEW

File: doc/ch04-md-contrvocabs.adoc [hunk 1]
```diff
@@ -68,6 +68,10 @@ data. This is primarily Sentinel datasets.
 
 |NySMAC |Datasets from Ny-Ålesund in the context of NySMAC (Ny-Ålesund Science Managers Committee).
 
+|KSS |Datasets from the Norsk klimaservicesenter (Norwegian Centre for Climate Services).
```
Comment (mortenwh) [inline doc/ch04-md-contrvocabs.adoc:71]: I wouldn't mix languages - what about the following?
```suggestion
|KSS |Datasets from the Norwegian Centre for Climate Services (no: Norsk klimaservicesenter).
```

File: doc/ch04-md-contrvocabs.adoc [hunk 2]
```diff
@@ -68,6 +68,10 @@ data. This is primarily Sentinel datasets.
...
+
```
Comment (mortenwh) [inline doc/ch04-md-contrvocabs.adoc:72]: perhaps provide the link to Geonorge?

File: thesauri/mmd-vocabulary.ttl [hunk 3]
```diff
@@ -1072,8 +1072,9 @@
...
...s/NMDC>, <https://vocab.met.no/mmd/Collection_Keywords/SIOS>, <https://vocab.met.no/mmd/Collection_Keywords/NSDN>, <https://vocab.met.no/mmd/Collection_Keywords/DOKI>, <https://vocab.met.no/mmd/Collection_Keywords/DAM>, <https://vocab.met.no/mmd/Collection_Keywords/ACCESS>, <https://vocab.met.no/mmd/Collection_Keywords/NBS>, <https://vocab.met.no/mmd/Collection_Keywords/APPL>, <https://vocab.met.no/mmd/Collection_Keywords/YOPP>, <https://vocab.met.no/mmd/Collection_Keywords/METNCS>, <https://vocab.met.no/mmd/Collection_Keywords/SESS2018>, <https://vocab.met.no/mmd/Collection_Keywords/SESS2019>, <https://vocab.met.no/mmd/Collection_Keywords/SESS2020>, <https://vocab.met.no/mmd/Collection_Keywords/SESS2022>, <https://vocab.met.no/mmd/Collection_Keywords/SIOSCD>, <https://vocab.met.no/mmd/Collection_Keywords/SIOSAP>, <https://vocab.met.no/mmd/Collection_Keywords/SIOSIN>, <https://vocab.met.no/mmd/Collection_Keywords/CVL>, <https://vocab.met.no/mmd/Collection_Keywords/AeN>, <https://vocab.met.no/mmd/Collection_Keywords/TONE>, <https://vocab.met.no/mmd/Collection_Keywords/NySMAC>, <https://vocab.met.no/mmd/Collection_Keywords/KSS>, <https://vocab.met.no/mmd/Collection_Keywords/GEONOR> .
```
Comment (mortenwh) [inline thesauri/mmd-vocabulary.ttl:1077]: Seems you forgot KSS?

Comment (mortenwh) [reply thesauri/mmd-vocabulary.ttl:1077]: no, I see it now.. To help readability, then - I suggest a line shift between each entry

File: thesauri/mmd-vocabulary.ttl [hunk 4]
```diff
@@ -1211,6 +1212,16 @@
   rdfs:seeAlso <https://nyalesundresearch.no/nysmac/> ;
   skos:definition "Datasets from Ny-Ålesund in the context of NySMAC (Ny-Ålesund Science Managers Committee)."@en .
 
+<https://vocab.met.no/mmd/Collection_Keywords/KSS>
+  a skos:Concept ;
+  skos:prefLabel "KSS"@en ;
+  rdfs:seeAlso <https://klimaservicesenter.no/> ;
+  skos:definition "Datasets from the Norsk klimaservicesenter (Norwegian Centre for Climate Services)."@en .
```
Comment (mortenwh) [inline thesauri/mmd-vocabulary.ttl:1219]: Same as above:
```suggestion
  skos:definition "Datasets from the Norwegian Centre for Climate Services."@en .
  skos:definition "Datasett fra Norsk klimaservicesenter."@no .
```

File: thesauri/mmd-vocabulary.ttl [hunk 5]
```diff
@@ -1211,6 +1212,16 @@
...
+
+<https://vocab.met.no/mmd/Collection_Keywords/GEONOR>
+  a skos:Concept ;
+  skos:prefLabel "GEONOR"@en ;
+  skos:definition "Datasets relevant for Geonorge."@en .
```
Comment (mortenwh) [inline thesauri/mmd-vocabulary.ttl:1224]:
```suggestion
  skos:definition "Datasets that shall be findable at Geonorge (https://geonorge.no)."@en .
```

File: thesauri/mmd-vocabulary.xml [hunk 6]
```diff
@@ -1602,6 +1603,21 @@
       </skos:Concept>
     </skos:member>
 
+    <skos:member>
+      <skos:Concept rdf:about="https://vocab.met.no/mmd/Collection_Keywords/KSS">
+        <skos:prefLabel xml:lang="en">KSS</skos:prefLabel>
+        <rdfs:seeAlso rdf:resource="https://klimaservicesenter.no/"/>
+        <skos:definition xml:lang="en">Datasets from the Norsk klimaservicesenter (Norwegian Centre for Climate Services).</skos:definition>
```
Comment (mortenwh) [inline thesauri/mmd-vocabulary.xml:1610]: same language confusion here

File: thesauri/mmd-vocabulary.xml [hunk 7]
```diff
@@ -1602,6 +1603,21 @@
...
+      </skos:Concept>
+    </skos:member>
+
+    <skos:member>
+      <skos:Concept rdf:about="https://vocab.met.no/mmd/Collection_Keywords/GEONOR">
+        <skos:prefLabel xml:lang="en">GENORG</skos:prefLabel>
+        <skos:definition xml:lang="en">Datasets relevant for Geonorge</skos:definition>
```
Comment (mortenwh) [inline thesauri/mmd-vocabulary.xml:1617]:
```suggestion
        <skos:definition xml:lang="en">Datasets that shall be findable at Geonorge (https://geonorge.no)</skos:definition>
```

**Review by mortenwh — CHANGES_REQUESTED** [review]
A few suggestions that might improve it a bit :)

**Comment by ferrighi** [comment]
Thanks, I'll fix and merge.

**Pull request merged**